### PR TITLE
Add functions to cast as built-in functions

### PR DESCRIPTION
**Pull request by k1LoW**
# Purpose

Add functions for casting as built-in functions to allow flexible use of values.

## Added functions

- `string` ... move internally configured as a built-in function
- `int` ... 🆕 
- `bool` ... 🆕

### CODE REVIEW

**Comment by k1LoW** [comment]
@k2tzumi If you have any feedback, I'd love to hear it!

File: README.md [hunk 1]
```diff
@@ -624,6 +624,9 @@ See [Language Definition](https://github.com/antonmedv/expr/blob/master/docs/Lan
 ### Built-in functions
 
 - `urlencode` ... [url.QueryEscape](https://pkg.go.dev/net/url#QueryEscape)
+- `string` ... [cast.ToString](https://pkg.go.dev/github.com/spf13/cast#ToString)
+- `int` ... [cast.ToInt](https://pkg.go.dev/github.com/spf13/cast#ToInt)
+- `bool` ... [cast.ToBool](https://pkg.go.dev/github.com/spf13/cast#ToBool)
```
Comment (k2tzumi) [inline README.md:629]: Thank you for maintaining the document!

File: operator.go [hunk 2]
```diff
@@ -927,7 +927,6 @@ func (o *operator) stepName(i int) string {
 
 func (o *operator) expand(in interface{}) (interface{}, error) {
 	store := o.store.toMap()
-	store["string"] = func(in interface{}) string { return fmt.Sprintf("%v", in) }
```
Comment (k2tzumi) [inline operator.go:930]: I didn't know there was such a thing as a string function. I just noticed. 😅 
I think it's a good feeling to have it officially implemented. 👍

**Review by k2tzumi — APPROVED** [review]
LGTM!

**Pull request merged**